packaging/rpm-ostree.spec

```diff
@@ -132,12 +138,20 @@ Requires:       librepo%{?_isa} >= %{librepo_version}
 # rpm-ostree wraps more of ostree (such as `ostree admin unlock` etc.)
 Requires: ostree
 Requires: bubblewrap
+# We have been building with fuse but changed to fuse3  on:
+# https://src.fedoraproject.org/rpms/rpm-ostree/c/3c602a23787fd2df873c0b18df3133c9fec4b66a
+# However our code is just calling fuse's fusermount.
+# We are updating our spec and code based on the discusion on:
+# https://github.com/coreos/rpm-ostree/pull/5047
+%if 0%{?el} <= 9
```

Comment:
I don't think this conditional is right...I suspect it's evaluating to true on fedora too and that's why we're hitting this bug still.

I *think* it may need to be e.g.

```
%if 0%{?rhel} && 0%{?el} <= 9
```
or so?

Reply:
https://github.com/search?q=+0%25%7B%3Fel%7D&type=code

I see people doing:

```%if 0%{?el} >= 6 || 0%{?rhel} >= 6```

Reply:
ok, trying your conditional in this push.

Reply:
Apparently 0%{?rhel}  should match the family of distros based on this: https://docs.fedoraproject.org/en-US/packaging-guidelines/DistTag/#_conditionals I wonder if the el conditional is throwing it.

Reply:
> %if 0%{?fedora} || 0%{?rhel} > 9 || 0%{?el} > 9

Check on `centos stream 9`, get `%{?rhel}` -> 9, but get `%{?el}` null. IMU, the condition LGTM.
```
$ podman run --rm -it quay.io/centos/centos:stream9 bash
[root@b7210efdfe5a /]# rpm --eval %{?rhel}
9
[root@b7210efdfe5a /]# rpm --eval %{?el}
```